### ISSUE TITLE
Thumbnails not showing up, but images are loaded

### ISSUE THREAD

**Issue by Zenahr** [nanostudio-org/nanogallery2#317]
# Environment

Windows 10
tested with Firefox and Chrome

# Description

The lightbox renders images correctly, but the overview doesn't show anything.

![image](https://user-images.githubusercontent.com/47085752/103113235-206dcd80-465a-11eb-93c4-b8d4d9fe3d62.png)
(overview)

![image](https://user-images.githubusercontent.com/47085752/103113259-409d8c80-465a-11eb-915a-3af6cbbc1437.png)
(single-image view)

This is my current code:

```html

<html>
  <head>
    <meta name="viewport" content="user-scalable=no, width=device-width, initial-scale=1, maximum-scale=1">

    <script type="text/javascript" src="https://cdn.jsdelivr.net/npm/jquery@3.3.1/dist/jquery.min.js"></script>

    <link href="https://cdn.jsdelivr.net/npm/nanogallery2@3/dist/css/nanogallery2.min.css" rel="stylesheet" type="text/css">
    <script type="text/javascript" src="https://cdn.jsdelivr.net/npm/nanogallery2@3/dist/jquery.nanogallery2.min.js"></script>

  </head>
  <body>

    <h1>Gallery made with nanogallery2</h1>

    <div ID="ngy2p" data-nanogallery2='{
        "itemsBaseURL": ".\assets\gallery\",
        "thumbnailWidth": "200",
        "thumbnailLabel": {
          "position": "overImageOnBottom"
        },
        "thumbnailAlignment": "center",
        "thumbnailOpenImage": true
      }'>
      
      <a href=".\assets\gallery\snap-2020-07-29-22h43m21s220.png" data-ngthumb=".\assets\gallery\snap-2020-07-29-22h43m21s220.png" data-ngdesc="">
        <!-- .\assets\gallery\snap-2020-07-29-22h43m21s220.png -->
      </a>
      
      <a href=".\assets\gallery\snap-2020-07-29-22h43m23s005.png" data-ngthumb=".\assets\gallery\snap-2020-07-29-22h43m23s005.png" data-ngdesc="">
        <!-- .\assets\gallery\snap-2020-07-29-22h43m23s005.png -->
      </a>
      
      <a href=".\assets\gallery\snap-2020-07-30-01h59m55s629.png" data-ngthumb=".\assets\gallery\snap-2020-07-30-01h59m55s629.png" data-ngdesc="">
        <!-- .\assets\gallery\snap-2020-07-30-01h59m55s629.png -->
      </a>
      
      <a href=".\assets\gallery\snap-2020-08-01-12h33m52s084.png" data-ngthumb=".\assets\gallery\snap-2020-08-01-12h33m52s084.png" data-ngdesc="">
        <!-- .\assets\gallery\snap-2020-08-01-12h33m52s084.png -->
      </a>
      
      <a href=".\assets\gallery\snap-2020-08-01-13h07m23s808.png" data-ngthumb=".\assets\gallery\snap-2020-08-01-13h07m23s808.png" data-ngdesc="">
        <!-- .\assets\gallery\snap-2020-08-01-13h07m23s808.png -->
      </a>
      
    </div>
    
  </body>
</html>
```

# Additional Notes

If I remove the full path from the `hrefs` the images do not load at all.

**Comment by Kris-B** [comment]
Could you please create a codepen or a jsfiddle to reproduce your issue?

**Comment by Zenahr** [comment]
@Kris-B well it works in codepen with externally loaded images via a cdn. But I'm having problems with locally hosted images inside a folder like `assets`.

**Comment by Zenahr** [comment]
@Kris-B I've made a repository for you to check out. It's all ready and set. I think that's the best way to reproduce the issue since this reflects my development setup correctly.

https://github.com/Zenahr/temp-nano2-debug

**Comment by Kris-B** [comment]
Thanks for the repo. I could find the issue: backslashes in paths when used in CSS.
fixed in v3.0.5

**Comment by Zenahr** [comment]
@Kris-B meaning without updating all I have to do is having `./assets/gallery/snap-2020-08-01-12h33m52s084.png` instead of `.\assets\gallery\snap-2020-08-01-12h33m52s084.png`?
